src/adapters/livekit.ts

```diff
@@ -0,0 +1,112 @@
+import { IBaseComponent, START_COMPONENT, STOP_COMPONENT } from '@well-known-components/interfaces'
+import { AppComponents } from '../types'
+import { AccessToken } from 'livekit-server-sdk'
+import { DataPacketKind, RemoteParticipant, Room, RoomEvent } from '@livekit/rtc-node'
+import { fromLivekitReceivedData } from '../logic/message-routing'
+
+export type ILivekitComponent = IBaseComponent
+
+export async function createLivekitComponent(
+  components: Pick<AppComponents, 'config' | 'logs' | 'messageRouting' | 'metrics'>
+): Promise<ILivekitComponent> {
+  const { config, logs, messageRouting, metrics } = components
+  const logger = logs.getLogger('livekit')
+
+  const [host, apiKey, apiSecret, roomName, identityPrefix, replicaId] = await Promise.all([
+    config.requireString('LIVEKIT_HOST'),
+    config.requireString('LIVEKIT_API_KEY'),
+    config.requireString('LIVEKIT_API_SECRET'),
+    config.requireString('LIVEKIT_ROOM_NAME'),
+    config.getString('LIVEKIT_IDENTITY_PREFIX'),
+    config.getString('REPLICA_NUMBER')
+  ])
+
+  const identity = `${identityPrefix}-${replicaId}`
```

Comment:
What about if one of them or both are nulls?

Reply:
What would you use as a default for the prefix? Name of the service? It's just semantics